Formula/lib/libmsquic.rb

```diff
@@ -0,0 +1,45 @@
+class Libmsquic < Formula
+  desc "Cross-platform, C implementation of the IETF QUIC protocol"
+  homepage "https://github.com/microsoft/msquic"
+  url "https://github.com/microsoft/msquic.git",
+  tag:      "v2.4.4",
+  revision: "e87ae9a37bcebdf5ab3725e5d882ddb50e4dbbb4"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "openssl@3"
+
+  def install
+    system "cmake", "-B", "build/openssl3",
+    "-DQUIC_OUTPUT_DIR=../../../artifacts/bin/Release_openssl3",
+    "-DCMAKE_BUILD_TYPE=Release", "-DQUIC_TLS=openssl3",
+    "-DQUIC_USE_SYSTEM_LIBCRYPTO=true", "-DQUIC_BUILD_PERF=off",
+    "-DQUIC_BUILD_TOOLS=off"
```

Comment:
Why not use `std_cmake_args`? It provides proper variables and catches FetchContent uses

Reply:
Not really sure what `std_cmake_args` contains, and how to use it, but I'll look it up through current formulas.

Reply:
I see what you mean now
https://github.com/Homebrew/brew/blob/da3b84b46a7fb61442da3c9af345fbe3b8aae034/Library/Homebrew/formula.rb#L1813
Thanks, I'll update this accordingly.

Reply:
One question: Why we need to avoid `FetchContent` usages? Looks like msquic uses it to fetch `OpenSSLQuic`, and workaround is probably setting `HOMEBREW_ALLOW_FETCHCONTENT`.